modules/new_user/new_user.py

```diff
@@ -0,0 +1,44 @@
+import os
+from discord.utils import get
+
+def is_new(member):
+    #if the member has the unassigned role
+    if get(member.roles, id = int(os.getenv('UNASSIGNED_ROLE_ID'))) != None:
+        return True
+    return False
+
+
+async def add_role(ctx, machon, year):
+    member = ctx.author
+    
+    #formatting role for env file
+    role_id = machon.upper() + '_YEAR_' + year + "_ROLE_ID" #format example: LEV_YEAR_1_ROLE_ID
+    new_role = get(member.guild.roles, id = int(os.getenv(role_id)))
```

Comment:
Does this include alumni?

Also, will need to get a copy of the new env variables for Heroku before merging.

Reply:
Maybe we shouldnt put the role IDs in the .env file. it doesn't matter if they're public, and we wouldn't need to keep updating our local copies whenever the file is changed. Consider putting it in a JSON, CSV, or TXT file instead.

Reply:
alumni joining are too rare to include in the bot, we'll handle that manually. If they didnt join while in machon, doubt they would join after theyre gone